### PR TITLE
BIB-18 added sonar config

### PR DESCRIPTION
**Pull request by Valeriy-V**

### CODE REVIEW

File: src/main/resources/sonarcloud/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# SonarCloud
+
+SonarQube analyzes branches and Pull Requests so you spot and resolve issues BEFORE you merge to main.
+
+### To set up SonarCloud locally run the following command :
+
+* install SonarLint plugin
+* go to Preferences -> Tools -> SonarLint
+* tick Bind Project to SonarQube/SonarCloud
```
Comment (EEM86) [inline src/main/resources/sonarcloud/README.md:9]: Is it correct word? `tick`

File: src/main/resources/sonarcloud/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* go to Configure connection
+* press + to create new connection
+* choose sonarCloud and enter any name 
+* In the next step enter the token (provided in slack) and add a new connection 
+* go back to SonarLint -> Project Settings and choose your connection from the dropdown list 
+* enter 'maingroon_svydovets-bring' as Project_key and apply changes
```
Comment (EEM86) [inline src/main/resources/sonarcloud/README.md:15]: The project name is not correct

File: src/main/resources/sonarcloud/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### Run SonarCloud locally
+
+In the bottom action bar you should see a new tab called SonarLint otherwise just search SonarLint through the IDE search     
+In SonarLint there's a Report tab where you can Analyse All Project Files
```
Comment (EEM86) [inline src/main/resources/sonarcloud/README.md:20]: Are we really nead this readme? Let CI github action does it. No need to run it manually, as I think

File: src/main/resources/sonarcloud/README.md (hunk 1 above)
Comment (Valeriy-V) [reply src/main/resources/sonarcloud/README.md:9]: Yep 😁

File: src/main/resources/sonarcloud/README.md (hunk 2 above)
Comment (Valeriy-V) [reply src/main/resources/sonarcloud/README.md:15]: oh

File: src/main/resources/sonarcloud/README.md (hunk 3 above)
Comment (Valeriy-V) [reply src/main/resources/sonarcloud/README.md:20]: maybe someone wants to verify it locally

Comment (azarot) [reply src/main/resources/sonarcloud/README.md:20]: I think there is no harm to keep this readme.

**Review by azarot — APPROVED** [review]

**Review by EEM86 — APPROVED** [review]

**Pull request merged**